common/genes/05_genes_special_accessories_clothes.txt

```diff
@@ -1361,6 +1361,18 @@ special_genes = {
 				boy = male
```

Comment:
Not sure what happened here, maybe UVs broken.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120005809-47557480-bfe9-11eb-9f41-8b52a53120d3.png)

</details>